### PR TITLE
added P in amongus

### LINKED ISSUE #806: Add character: <P> to font: <amongus>
*Character* : <P>
*Font* : <amongus>

### PR DESCRIPTION
**Pull request by Sukriti-Rai**
*Describes your contribution*

### added P in amongus

closes #806

### CODE REVIEW

File: Fonts/amongus/amongus.h [hunk 1]
```diff
@@ -213,16 +213,21 @@ class Amongus : public Fonts
...
         {
-            char **character = getCharGrid(ROWS,COLS);
+            char **character = getCharGrid(4,6);
 
             //Enter the character grid in ROWS X COLS
-
+            character[0][1]= character[0][2]=character[0][3]=character[0][4]=character[1][3]=character[2][2]=character[2][3]=character[2][4]=character[3][1]=character[3][2]='_';
+            character[1][0]=character[3][0]='(';
+            character[2][1]=character[3][3]=')';
+            character[1][5]='\';
```
Comment (AnshumanMahato) [inline Fonts/amongus/amongus.h:225]: @Sukriti-Rai, resolve this error
image

Replace '\' with '\\'

Comment (AnshumanMahato) [inline Fonts/amongus/amongus.h:225]: @Sukriti-Rai, resolve this error
image

Replace '\' with '\\'

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/amongus/amongus.h (hunk 1 above)
Comment (Sukriti-Rai) [reply Fonts/amongus/amongus.h:225]: what changes to commit..?

Comment (AnshumanMahato) [reply Fonts/amongus/amongus.h:225]: @Sukriti-Rai, replace the '\\" with "\\\\"

**Pull request merged**